v-spec.adoc

```diff
@@ -4355,6 +4355,11 @@ register, and can copy whole vector register groups.
 NOTE: These instructions are intended to aid compilers to shuffle
 vector registers without needing to know or change `vl` or `vtype`.
 
+NOTE: The move is performed even if `vstart` {ge} `vl`.
+
+NOTE: Even though these instructions ignore the `vtype` setting,
+they still raise an illegal-instruction exception if `vill`=1.
```

Comment:
> NOTE: Even though these instructions ignore the `vtype` setting,
> they still raise an illegal-instruction exception if `vill`=1.
> 

I would argue against this.
vill = 1 only indicates that the requested vtype in invalid. 
(It doesn't disable the vector unit we have other means for that)
Any instructions that rely on a valid vtype are therefore invalid instructions when vill is set. Instructions that don't rely on vtype thus are not required to be invalid when vill = 1.
Consider the 64 bit encoding of which this V-extension is intended to be a "compressed encoding".
It is expected that the vtype will be incorporated into the instruction.
The whole register move instructions then would have the vtype field(s) absent/repurposed/reserved(except for default value) and would not support dynamic vtype (which is the default for our 32 bit encoding).
Thus the whole move instructions should ignore vtype completely.